crates/red_knot_workspace/src/workspace.rs

```diff
@@ -79,6 +81,10 @@ pub struct Workspace {
     /// The (first-party) packages in this workspace.
     #[return_ref]
     package_tree: BTreeMap<SystemPathBuf, Package>,
+
+    /// The unresolved search path configuration.
+    #[return_ref]
+    pub search_path_configuration: SearchPathConfiguration,
```

Comment:
We may want to store the entire `WorkspaceConfiguration` on `Workspace` long term but it isn't clear to me if we actually want to do this. That's why I went with storing the absolute minimum